lib/link/messages.js

```diff
@@ -294,6 +294,7 @@ messages.listInstances = new Request({
 					"name": { type: "string" },
 					"id": { type: "integer" },
 					"assigned_slave": { type: ["null", "integer"] },
+					"status": { enum: ["stopped", "initialized", "running"] },
```

Comment:
This should be added to the required list a few lines up in the schema as well.  I've usually added the type parameter for enum validated items, though I see the updateInstances event has it without `type: "string"` as well.